.azure-pipelines/get_dut_version.py

```diff
@@ -19,13 +20,34 @@
 RC_GET_DUT_VERSION_FAILED = 2
 
 
-def get_duts_version(sonichosts):
+def get_duts_version(sonichosts, output=None):
     try:
         ret = {}
         duts_version = sonichosts.command("show version")
         for dut, version in duts_version.items():
-            ret[dut] = version["stdout_lines"]
-        return ret
+            ret[dut] = {}
+            dut_version = version["stdout_lines"]
+
+            for line in dut_version:
+                if ":" in line:
+                    key = line.split(":", 1)[0].strip()
```

Comment:
no need to split it twice, it will generate 2 exactly the same array which will waste the memory:
`
line_splitted = line_splitted 
key = line_splitted[0].strip()
value = line_splitted[0].strip()
`